db/seeds/dev.rb

```diff
@@ -24,3 +25,58 @@
     CoreBoxItem.joins(:abuse_type, :inventory_type).where(abuse_type: abuse_type, inventory_type: inventory_type).first_or_create!
   end
 end
+
+40.times do
+  is_underage = [{boolean: false, value: "12+"}, { boolean: true, value: "UNDERAGE"}].sample
+
+  requester = Requester.create!(
```

Comment:
Hello! I have an unsolicited suggestion :) 
You can use the Factory from `spec/factories/requester.rb`, that way all the faker logic only has to live in one place if it needs updates

Reply:
Yeah, I thought about that after I created a PR. 
I will work on this and the original issue (seeds for other tables) later this month :)